src/components/HeaderHome/HeaderHome.tsx

```diff
@@ -0,0 +1,22 @@
+import "./HeaderHome.scss";
+
+type HeaderHomeProps = {
+  image: string;
+  date?: string;
```

Comment:
I think that these should not be optional or if they are optional you need to conditionally render the img and p tag related to them.